app/form_models/jobseekers/professional_body_membership_form.rb

```diff
@@ -0,0 +1,8 @@
+class Jobseekers::ProfessionalBodyMembershipForm
+  include ActiveModel::Model
+
+  attr_accessor :name, :membership_type, :membership_number, :date_membership_obtained, :exam_taken
```

Comment:
do we need this abstraction, or could we attach the validations directly to the model?